libqtile/backend/wayland/window.py

```diff
@@ -372,6 +385,22 @@ def paint_borders(self, color: ColorsType, width) -> None:
                 self.bordercolor = [_rgb(color)]
         self.borderwidth = width
 
+    def add_idle_inhibitor(
+        self, surface: surface.Surface, _x: int, _y: int, inhibitor: IdleInhibitorV1 | None
```

Comment:
How come the type of inhibitor is `IdleInhibitorV1 | None` and not just `IdleInhibitorV1`?

Reply:
Because the type signature for all implementations of for_each_surface is `data: T | None` so without that mypy complains.

Reply:
OK cool, thanks